src/transformers/configuration_utils.py

```diff
@@ -81,6 +81,15 @@ class PretrainedConfig(PushToHubMixin):
       model.
     - **num_hidden_layers** (`int`) -- The number of blocks in the model.
 
+    <Tip warning={true}>
+
+    Setting parameters for sequence generation in the model config is deprecated. For backward compatibility, loading
+    some of them will still be possible, but attempting to overwrite them will throw an exception -- you should set
+    them in a [~transformers.GenerationConfig]. Check the documentation of [~transformers.GenerationConfig] for more
+    information about the individual parameters.
+
+    </Tip>
```

Comment:
Instead of documenting `generate` parameters in `PretrainedConfig`, which is deprecated, show a warning in the docs pointing towards `GenerationConfig`